### PR TITLE
Add conversion hPA/inHG, kPa/PSI

### PR DESCRIPTION
**Pull request by zuckschwerdt**
- add automatic conversion hPA/inHG, kPa/PSI
- extends automatic conversion to all C/F fields
- minor style updates

### CODE REVIEW

**Comment by zuckschwerdt** [comment]
I can already do it this time ;)

File: include/util.h [hunk 1]
```diff
@@ -12,6 +12,7 @@
 #define INCLUDE_UTIL_H_
 
 #include <stdint.h>
+#include <stdbool.h>
```
Comment (merbanan) [inline include/util.h]: Is this needed for this or leftovers from the bool patch ?

**Comment by zuckschwerdt** [comment]
It's needed for `bool str_endswith()` (util.h/.c) -- fits well and C99 supports it, but could also return an int.

**Comment by merbanan** [comment]
Ok, all good then.

**Comment by merbanan** [comment]
Can we merge this ?

**Pull request merged**